.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2.1
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+       - image: circleci/ruby:3.1.2
```

Comment:
I'm using this locally, so I updated to this, though we could use the min supported ruby version here (2.5)